nerfstudio/data/datasets/base_dataset.py

```diff
@@ -115,3 +116,14 @@ def get_metadata(self, data: Dict) -> Dict:
     def __getitem__(self, image_idx: int) -> Dict:
         data = self.get_data(image_idx)
         return data
+
+    def get_image_filenames(self) -> List[Path]:
```

Comment:
This could be a property, https://www.programiz.com/python-programming/property

If it's a property, you can remove the `Args` and `Returns`

Reply:
Good idea.